core/clients/file_storage/classes/local_storage.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import pathlib
+import os
+
+from dev.yeti.core.clients.file_storage.classes.interface import FileStorageClient
+
+class LocalStorageClient(FileStorageClient):
+    PREFIX = ""
+
+    def __init__(self, path: str):
+        self.path = pathlib.Path(path)
+        self.path.mkdir(parents=True, exist_ok=True)
+
+        logging.info(f"Initialized local storage client with path {self.path}")
+
+    def __file_path(self, file_name: str) -> pathlib.Path:
```

Comment:
```suggestion
    def _file_path(self, file_name: str) -> pathlib.Path:
```

Reply:
Single underscore for private functions

Reply:
Fixed in [87f453e](https://github.com/yeti-platform/yeti/pull/1137/commits/87f453e6582f0f010708b23d8e94a19e3b9493c2)